runtime/service/src/main/java/org/apache/polaris/service/config/ServiceProducers.java

```diff
@@ -139,10 +139,23 @@ public RealmConfig realmConfig(CallContext callContext) {
     return callContext.getRealmConfig();
   }
 
+  @Produces
+  @ApplicationScoped
+  public PolarisAuthorizerFactory polarisAuthorizerFactory(
+      AuthorizationConfiguration authorizationConfig,
+      @Any Instance<PolarisAuthorizerFactory> authorizerFactories) {
+    PolarisAuthorizerFactory factory =
+        authorizerFactories
+            .select(Identifier.Literal.of(authorizationConfig.type().getValue()))
```

Comment:
Is it worth having an enum for the type? `PolarisAuthorizerFactory` are pluggable at build time. There is no way for `runtime/service` code to be in sync will all possible downstream projects that want to customize `PolarisAuthorizerFactory` 🤔

Reply:
That's an interesting point. I can remove this

Reply:
yeah makes sense - I've already made this change @dimas-b